SCJ.Booking.sln

```diff
@@ -21,6 +21,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SCJ.Booking.Data", "databas
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SCJ.Booking.TaskRunner", "taskrunner\SCJ.Booking.TaskRunner.csproj", "{C9827C16-2A98-4117-AC6E-B3723CCC25E7}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "config", "config", "{9D976C8E-DCDD-4599-8ECF-5038232813FE}"
+	ProjectSection(SolutionItems) = preProject
+		.env = .env
+		.env.example = .env.example
+	EndProjectSection
+EndProject
```

Comment:
Assuming this is something for Visual Studio? Or does it affect the server in some way?

Reply:
This just puts the .env files into a solution folder so they are accessible from the solution explorer inVisual Studio